h/static/scripts/directive/test/annotation-test.js

```diff
@@ -1460,137 +1353,42 @@ describe('annotation.js', function() {
         annotation.$update = sandbox.stub().returns(Promise.resolve());
         var controller = createDirective(annotation).controller;
         controller.edit();
-        controller.save();
-
-        // The controller currently removes the draft whenever an annotation
-        // update is committed on the server. This can happen either when saving
-        // locally or when an update is committed in another instance of H
-        // which is then pushed to the current instance.
-        annotation.updated = (new Date()).toISOString();
-        $scope.$digest();
-        assert.calledWith(fakeDrafts.remove, annotation);
+        return controller.save().then(function() {
+          assert.calledWith(fakeDrafts.remove, annotation);
+        });
       });
     });
 
-    describe('when the focused group changes', function() {
-      it('updates the current draft', function() {
+    describe('onGroupFocused()', function() {
```

Comment:
I actually preferred the description of this method as "when the focused group changes". The fact that this event is handled by a function called `onGroupFocused()` internally is an implementation detail.